### DISCUSSION TITLE
Updating user profile field

### DISCUSSION THREAD

**Discussion by ashermyers** [pocketbase/pocketbase#968] [Q&A] [answered]
How would I update a user profile field using the javascript api? This is my code but its not working:

`const updated = await client.users.update("<userid>", {
            DOB:"12/14/03"
        });
`

**Comment by ganigeorgiev** [comment, marked as answer]
In PocketBase v0.7.x users and profiles are 2 separate things. Users have fixed fields (eg. email, verified, etc), while the `profiles` is a regular collection and can have any custom fields. I guess that `DOB` from your example is a `profiles` field, right?
If that's the case then you have to send the request to the `profiles` collection using the `client.records` service using the profile id, not the `client.users` one:

```js
const updated = await client.records.update("profiles", "YOUR_PROFILE_ID", { DOB:"12/14/03" });
```
You can get your profile Id from your user model - `user.profile.id` (or from `client.authStore.model.profile.id` if authenticated).

---

Please note that this is no longer the case for PocketBase v0.8.0-rc because users and profiles are now merged per #376. More info about the changes could be found in https://github.com/pocketbase/pocketbase/releases/tag/v0.8.0-rc1.
If you decide to migrate to PocketBase v0.8.0 you have to use the pre-release of the JS SDK - https://www.npmjs.com/package/pocketbase/v/0.8.0-rc2.
Stable v0.8.0 will be available in the next 1-2 weeks.
